nexus/types/src/external_api/params.rs

```diff
@@ -1109,6 +1115,12 @@ pub struct FloatingIpCreate {
     /// The parent IP pool that a floating IP is pulled from. If unset, the
     /// default pool is selected.
     pub pool: Option<NameOrId>,
+
+    /// Preferred IP version when allocating from the default pool.
```

Comment:
All the doc comments look great to me. My only nitpick is that "preferred" here sounds like you're asking for it but might not get it. Is that true? E.g., will it fall back to the other pool if your "preferred" one is full? I think I would prefer it to fail in that case. And if not, maybe "IP version to use when allocating..." would sidestep the issue.

Same applies to other uses of "preferred".

Reply:
Good point. Will fix, as it does fail out (which is the right semantics).